SimPEG/simulation.py

```diff
@@ -482,20 +710,44 @@ def dpred(self, m=None, f=None):
 
 
 class LinearSimulation(BaseSimulation):
-    """
-    Class for a linear simulation of the form
+    r"""Linear forward simulation class.
+
+    The ``LinearSimulation`` class is used to define forward simulations of the form:
 
     .. math::
+        \mathbf{d} = \mathbf{G \, f}(\mathbf{m})
+
+    where :math:`\mathbf{m}` are the model parameters, :math:`\mathbf{f}` is a
+    mapping operator (optional) from the model space to a user-defined parameter space,
+    :math:`\mathbf{d}` is the predicted data vector, and :math:`\mathbf{G}` is an
+    ``(n_data, n_param)`` linear operator.
+
+    The ``LinearSimulation`` class is generally used as a base class that is inherited by
+    other simulation classes within SimPEG. However, it can be used directly as a
+    simulation class if the :py:attr:`G` property is used to set the linear forward
+    operator directly.
 
-        d = Gm
+    By default, we assume the mapping operator :math:`\mathbf{f}` is the identity map,
+    and that the forward simulation reduces to:
 
-    where :math:`d` is a vector of the data, `G` is the simulation matrix and
-    :math:`m` is the model.
-    Inherit this class to build a linear simulation.
+    .. math::
+        \mathbf{d} = \mathbf{G \, m}
+
+    Parameters
+    ----------
+    mesh : discretize.BaseMesh, optional
+        Mesh on which the forward problem is discretized. This is not necessarily
+        the same as the mesh on which the simulation is defined.
+    model_map : SimPEG.maps.BaseMap
+        Mapping from the model parameters to vector that the linear operator acts on.
+    G : (n_data, n_param) numpy.ndarray or scipy.sparse.csr_matrx
+        The linear operator. For a ``model_map`` that maps within the same vector space
+        (e.g. the identity map), the dimension ``n_param`` equals the number of model parameters.
+        If not, the dimension ``n_param`` of the linear operator will depend on the mapping.
     """
 
     linear_model, model_map, model_deriv = props.Invertible(
-        "The model for a linear problem"
+        "the model for a linear problem"
```

Comment:
I think elsewhere we start with a capital letter? So might be good to stick with that for consistency?